src/transformers/models/align/processing_align.py

```diff
@@ -58,19 +108,6 @@ def __call__(self, text=None, images=None, padding="max_length", max_length=64,
             images (`PIL.Image.Image`, `np.ndarray`, `torch.Tensor`, `List[PIL.Image.Image]`, `List[np.ndarray]`, `List[torch.Tensor]`):
                 The image or batch of images to be prepared. Each image can be a PIL image, NumPy array or PyTorch
                 tensor. Both channels-first and channels-last formats are supported.
-            padding (`bool`, `str` or [`~utils.PaddingStrategy`], *optional*, defaults to `max_length`):
-                Activates and controls padding for tokenization of input text. Choose between [`True` or `'longest'`,
-                `'max_length'`, `False` or `'do_not_pad'`]
-            max_length (`int`, *optional*, defaults to `max_length`):
-                Maximum padding value to use to pad the input text during tokenization.
-
-            return_tensors (`str` or [`~utils.TensorType`], *optional*):
```

Comment:
`return_tensors` is the only one I'm not sure we want to remove from the docstring, as it's something users have to specify if they want to be able to use the outputs for a model

Reply:
that's a good point, in general anything that is `common` could be here